### ISSUE TITLE
Lagging codes on WearOS

### ISSUE THREAD

**Issue by JVKeller** [jamie-mh/AuthenticatorPro#1135]
**Describe the bug**
Delay or lag in getting the newest code from phone after short idle time with Galaxy watch / Wear OS

**To Reproduce**
1. Log on the webapp
2. open Authenticator on watch
3. try to use code

**Expected behavior**
Should log on using the displayed token.

**App Version**
1.24.1 Phone
2.15.1 Wear OS

**Additional context**
The displayed code with be behind by two, or sometimes three, tokens. It will eventually cycle through to the current one, but the app or screen will have shut off by then.

**Comment by jamie-mh** [comment]
The codes on the watch are generated using the system clock. Do you have a time difference between your phone and the watch? That's the likeliest explanation.

Thanks

**Comment by JVKeller** [comment]
Nope, same time. My guess would be some cache or syncing timing. 

Some times when you open it, you'll see a code that will change after a few seconds, and sometimes the next code will only show 5-10 seconds, and then then third code is back on track.

**Comment by mrturkeyfarmer** [comment]
Can confirm a similar issue, when accessing the code from the tile screen, the code is often out of sync. 

If the app is opened to the code of choice, they always seem to be in sync

Galaxy Watch 5 Pro
Samsung Galaxy S23 Ultra
1.25.1 Phone
1.25.1 Wear OS

**Comment by rynshilla** [comment]
I use the tile feature all the time. If there's any testing I can help with, I'd be happy to. It seems to just not sync the updated token with the tile until _something_ happens in the background. If I tap the tile and select the account, then the updated code appears and seems to update the tile as well. I'd be happy to help in any way I can. ( Pixel watch 4, wear OS 6, but I've noticed it on pixel watch 1 and 2 as well.)
